airbyte-integrations/connectors/source-amplitude/acceptance-test-config.yml

```diff
@@ -1,24 +1,33 @@
 connector_image: airbyte/source-amplitude:dev
-tests:
+test_strictness_level: high
+acceptance_tests:
   spec:
-    - spec_path: "source_amplitude/spec.json"
-      backward_compatibility_tests_config:
-        disable_for_version: "0.1.17"
+    tests:
+      - spec_path: "source_amplitude/spec.json"
   connection:
-    - config_path: "secrets/config.json"
-      status: "succeed"
-    - config_path: "integration_tests/invalid_config.json"
-      status: "failed"
+    tests:
+      - config_path: "secrets/config.json"
+        status: "succeed"
+      - config_path: "integration_tests/invalid_config.json"
+        status: "failed"
   discovery:
-    - config_path: "secrets/config.json"
+    tests:
+      - config_path: "secrets/config.json"
   basic_read:
-    - config_path: "secrets/config.json"
-      configured_catalog_path: "integration_tests/streams_with_output_records_catalog.json"
+    tests:
+      - config_path: "secrets/config.json"
+        expect_records:
+          path: "integration_tests/expected_records.txt"
+        empty_streams:
+          - name: "events"
+            bypass_reason: "The stream is too large and frequently changing, due to test with live account."
```

Comment:
We must not use a live account! 
(cc @evantahler)

Reply:
Please follow this link: https://analytics.amplitude.com/airbyteinc/home to check out for yourself, whether or not we use the live account or test one.
Use the LastPass creds for `integration-test@airbyte.io`

Reply:
@alafanechere 
Are we freezing this one until we can use the sandbox?

Reply:
I confirm the Amplitude account is a live account for airbyte on which we have the `integration-test-@airbyte.io` user.
I believe the `expected_records.json` file is not containing sensitive data at the moment but I'd like a double-check from @evantahler . 
In the short term, I think it's OK to keep the live account for testing but in long run, the test might fail after unrelated actions on the Amplitude account.

Reply:
Access has been removed

Reply:
Information about the test account has been removed form  LastPass and GSM. SAT will fail until a new sandbox account & secrets are added.